src/main/java/org/sciborgs1155/robot/Hopper/RealHopper.java

```diff
@@ -0,0 +1,22 @@
+package org.sciborgs1155.robot.Hopper;
+
+import static org.sciborgs1155.robot.Hopper.HopperConstants.*;
+
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+
+public class RealHopper implements HopperIO {
+  CANSparkMax motor = new CANSparkMax(5, MotorType.kBrushless);
+  RelativeEncoder encoder = motor.getEncoder();
+
+  @Override
+  public double getSpeed() {
+    return motor.getEncoder().getVelocity() * WHEEL_RADIUS;
```

Comment:
why do you need to multiply by the wheel radius? That's important for something like drive, when what you care about is how much ground you're covering, but here it gets a little confusing. You need to make a clear decision (with the other rookies ideally) about what you want to do in m/s vs rps. For instance, I'm pretty sure shooter is in rps rn, so if that's the case, this really should be too.

Reply:
And make sure you make the units clear in the comments